From 0324729bbc8a76a26ea9d1bb1b2a1bccbf70b291 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Mon, 18 Aug 2025 14:55:11 +0200
Subject: [PATCH 11/55] include/kernel-6.1: add v6.1 hash

Adds kernel 6.1 hash to build system for QSDK-based targets using this kernel version.

Signed-off-by: John Crispin <john@phrozen.org>
---
 include/kernel-6.1 | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 include/kernel-6.1

diff --git a/include/kernel-6.1 b/include/kernel-6.1
new file mode 100644
index 0000000000..60aadf0300
--- /dev/null
+++ b/include/kernel-6.1
@@ -0,0 +1,2 @@
+LINUX_VERSION-6.1 = .31
+LINUX_KERNEL_HASH-6.1.31 = ca116439ce055924f367ed3ad6c41fd2d84bd961a6cc1795443d152aad36310b
-- 
2.34.1

